docs/manuals/core/scheduling/index.md

```diff
@@ -0,0 +1,278 @@
+
+- `datetime.datetime` object,
+- `datetime.time` object,
+- `datetime.timedelta` object,
+- or a string in one of the following formats: "%Y-%m-%d %H:%M:%S", "%Y-%m-%d %H:%M", "%Y-%m-%d",
+  "%H:%M:%S", "%H:%M".
+
+If only a time is provided, the date is set to today.
+
+```python linenums="1"
+from datetime import datetime, timedelta, time
+import taipy as tp
+from my_config import monthly_scenario_cfg
+
+monthly_scenario = tp.create_scenario(monthly_scenario_cfg)
+
+# Submit a scenario until a 18:30 today
+tp.Scheduler.every(1).hours.until("18:30").do(tp.submit, monthly_scenario)
+
+# Submit a scenario until a 2030-01-01 18:33
+tp.Scheduler.every(1).hours.until("2030-01-01 18:33").do(tp.submit, monthly_scenario)
+
+# Submit a scenario until today 11:33:42
+tp.Scheduler.every(1).hours.until(time(11, 33, 42)).do(tp.submit, monthly_scenario)
+
+# Submit a scenario for every hour until the next 8 hours
+tp.Scheduler.every(1).hours.until(timedelta(hours=8)).do(tp.submit, monthly_scenario)
+
+# Submit a scenario until a specific datetime
+tp.Scheduler.every(1).months.until(datetime(2030, 1, 1, 12, 30, 0)).do(tp.submit, monthly_scenario)
+```
+
+## Get all scheduled methods
+
+To retrieve all scheduled methods from the scheduler, use `tp.Scheduler.get_scheduled_methods()` method.
+
+```python linenums="1"
+import taipy as tp
+from my_config import monthly_scenario_cfg
+
+monthly_scenario = tp.create_scenario(monthly_scenario_cfg)
+
+tp.Scheduler.every().second.do(tp.submit, monthly_scenario)
+
+all_scheduled_methods = tp.Scheduler.get_scheduled_methods()
+```
+
+The `tp.Scheduler.get_scheduled_methods()` method returns a list of `ScheduledMethod^` objects, which you can use to
+get information or cancel the scheduled method.
+
+## Cancel scheduled methods
+
+To remove a scheduled method from the scheduler, use the `tp.Scheduler.cancel_scheduled_method(scheduled_method)` method.
+
+```python linenums="1"
+import taipy as tp
+from my_config import monthly_scenario_cfg
+
+monthly_scenario = tp.create_scenario(monthly_scenario_cfg)
+
+monthly_submit_method = tp.Scheduler.every().second.do(tp.submit, monthly_scenario)
+
+tp.Scheduler.cancel_scheduled_method(monthly_submit_method)
+```
+
+To remove all scheduled methods from the scheduler, use `tp.Scheduler.clear()`
+
+```python linenums="1"
+import taipy as tp
+from my_config import monthly_scenario_cfg
+
+monthly_scenario = tp.create_scenario(monthly_scenario_cfg)
+
+tp.Scheduler.every().second.do(tp.submit, monthly_scenario)
+
+tp.Scheduler.clear()
+```
+
+## Shortcut for scheduling Taipy methods
+
+Taipy `Scheduler^` provides shortcut for scheduling popular Taipy methods, including:
+
+- `taipy.create_scenario()`
+- `taipy.submit()`
+- A combination of `taipy.create_scenario()` and `taipy.submit()`
+
+Instead of calling the `do()` method and providing the Taipy method to schedule, you can call
+the following method directly from the scheduler.
```

Comment:
```suggestion
Instead of using the `do()` method and specifying the Taipy method to schedule, you can directly call the following method from the scheduler.
```